diff -ur gnuserv-3.12.6-vanilla/Makefile.in gnuserv-3.12.6/Makefile.in
--- gnuserv-3.12.6-vanilla/Makefile.in	2003-05-09 03:23:56.000000000 +0200
+++ gnuserv-3.12.6/Makefile.in	2004-04-16 23:18:34.484834144 +0200
@@ -55,11 +55,11 @@
 # data files.
 # As a special exception, see `infodir' and `includedir' below.
 datadir = $(prefix)/share
-
+mandir = $(datadir)/man
 elispdir = $(datadir)/emacs/site-lisp
 
 # Where to install the manual pages.
-man1dir = $(prefix)/man/man1
+man1dir = $(mandir)/man1
 # Prefix/suffix for each installed man page, normally empty or `g'.
 man1prefix = $(binprefix)
 man1suffix = $(binsuffix)
